app/javascript/gobierto_data/webapp/components/commons/Info.vue

```diff
@@ -22,6 +22,27 @@
         :label="labelSubject"
         :text="categoryDataset"
       />
+      <InfoBlockText
```

Comment:
@Crashillo @stbnrivas it would be good to show both source text and url as a link (with target blank), and not in two separated fields, don't you think?

Reply:
In order to do so, you should either create a new component or parametrize the current one

Reply:
yes, could you help @stbnrivas ?

Reply:
we already create a new component to show as a link <InfoBlockLink>. Also I opened a new issue with another way to do it.

#3830